functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,238 @@
+# inflate-helm-chart
+
+[helm]: https://helm.sh/
+[charts]: https://helm.sh/docs/topics/charts/
+[local]: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/inflate-helm-chart-local
+[remote]: https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/inflate-helm-chart-remote
+[helm template command]: https://helm.sh/docs/helm/helm_template/
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `inflate-helm-chart` function inflates a local or remote Helm chart. 
+
+<!--mdtogo-->
+
+Helm is a package manager for kubernetes that uses a packaging format
+called charts. A chart is a collection of files within a directory, which 
+contain templates, CRDs, values, and metadata. 
+
+This function renders charts locally by using the 
+[helm template command],
+so that helm charts can be inflated without needing to install the
+helm binary directly.
+
+You can learn more about helm [here][helm] and more about helm
+charts [here][charts].
+
+<!--mdtogo:Long-->
+
+## Usage
+
+This function can be used with any KRM function orchestrators such as kpt
+or kustomize to inflate a specified helm chart.
+
+In kpt, the function can only be run imperatively. The function either
+needs network access to inflate a remote chart or needs a local file to be mounted
+into the container to inflate a local chart. As a result, to run the
+function with `kpt fn eval`, the flag `--network` must be used for remote charts,
+and the flag `--mount` must be used for local charts. See the examples for inflating
+[local] and [remote] charts.
+
+It can be used declaratively when run with kustomize. To run the function with kustomize,
+the `network` field is needed for remote charts and the `mounts` field is needed for local charts.
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+There are 2 kinds of `functionConfig` supported by this function:
+
+- `ConfigMap`
+- A custom resource of kind `InflateHelmChart`
+
+#### `ConfigMap`
+To use a `ConfigMap` as the `functionConfig`, the desired parameters must be
+specified in the `data` field:
+
+```yaml
+data:
+  chartHome: string
+  configHome: string
+  name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesFile: string
+  includeCRDs: string
+```
+
+
+| Field        |  Description | Example
+| -----------: |  ----------- | -----------
+`chartHome`    | A filepath to a directory of charts. The function will look for the chart in this local directory before attempting to pull the chart from a specified repo. Defaults to "tmp/charts". When run in a container, this path MUST have the prefix "tmp/". | tmp/charts
```

Comment:
Should it be `/tmp/charts`?
Please audit the doc ensure consistency?

Reply:
Same as the above comment, I intentionally left it as `tmp/charts` because when using this as an exec function and inflating a local chart, this will allow the users to have the charts in their current directory by default. In a container environment, the `absPath()` method converts this to `/tmp/charts`.

If we want to use `/tmp/charts` by default instead, we can, but this will require kustomize users who use this as an exec function to specify an extra field for the path to their local charts.

Reply:
Sorry, I missed your earlier comment. There are a lot of comments here :P 

It seems your Dockerfile is using the default WORKDIR which is `/`. So it should be fine here. When in container, it resolves to `/tmp/charts`.